backend/Dockerfile

```diff
@@ -32,7 +32,8 @@ COPY backend/requirements.txt .
 RUN python3 -m pip install -r requirements.txt
 
 # Downloading Argo CLI so that the samples are validated
-ADD  https://github.com/argoproj/argo/releases/download/v2.3.0/argo-linux-amd64 /usr/local/bin/argo
+#ADD  https://github.com/argoproj/argo/releases/download/v2.7.5/argo-linux-amd64 /usr/local/bin/argo
```

Comment:
Can you add a comment here, why we need to use the old argo CLI?

Reply:
Oh, sure. I'll do that.
The reason is that the Argo CLI is only used for validation (argo lint) and later versions of Argo CLI can no longer work inside a container.